src/components/Datepicker/Datepicker.jsx

```diff
@@ -385,10 +385,22 @@ DatePicker.propTypes = {
     PropTypes.object,
     PropTypes.number,
   ]),
-  value: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
+  value: PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired,
   isRange: PropTypes.bool,
   lang: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
-  onChange: PropTypes.func,
+  onChange: function (props, propName) {
+    var fn = props[propName];
+    if (
+      !fn.prototype ||
```

Comment:
when `onChange` is `null` or `undefined`, the error message is:
```error
Warning: Failed prop type: Cannot read properties of undefined (reading 'prototype')
```

I think we should also display the `... must be a function with 2 (if isMonthsMode is on) or 3 args. Args are: year, month and day.` error.

Reply:
Yeah, will add that. thanks!